impeller/tessellator/tessellator.h

```diff
@@ -175,21 +175,22 @@ class Tessellator {
   /// @brief      Given a convex path, create a triangle fan structure.
   ///
   /// @param[in]  path  The path to tessellate.
+  /// @param[in]  host_buffer  The host buffer for allocation of vertices/index
+  ///                          data.
   /// @param[in]  tolerance  The tolerance value for conversion of the path to
   ///                        a polyline. This value is often derived from the
-  ///                        Matrix::GetMaxBasisLength of the CTM applied to the
-  ///                        path for rendering.
-  ///
-  /// @return A point vector containing the vertices in triangle strip format.
+  ///                        Matrix::GetMaxBasisLengthXY of the CTM applied to
+  ///                        the path for rendering.
+  /// @param[in]  line_strip if true, generates line strip geometry instead of a
```

Comment:
A line strip is a GL_LINE_STRIP geometry. How about a link to the primitive type?

Reply:
I don't understand why a tessellator would be generating a line strip, tessellation means converting something to packed polygons (triangles in our case).  I see, this is doing stuff like converting a bezier path to a line strip?

Yea, mentioning GL_LINE_STRIP would help to distinguish it from a line make with a triangle strip.  Especially since a generating a GL_LINE_STRIP isn't technically tessellation.

The brief for this method also explicitly says it creates a triangle fan structure, that should be updated.

Now that I think of it.  Don't you think this would be better off as it's own function instead of a bool parameter?  Generally it's advised to make procedures do 1 thing.  We can share the code in a helper function.

Reply:
Yeah I agree with you, a separate method would be better in this case. I suppose since its a line primitive its not exactly tessellation but primitive generation :)